ironic-deployment/overlays/e2e-with-inspector/ironic-patch.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ironic
+spec:
+  template:
+    spec:
+      containers:
+        - name: ironic-dnsmasq
```

Comment:
Sorry, missed this last time. This needs to be indented properly as well.

Reply:
Done